freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientExceptionUtil.java

```diff
@@ -176,6 +176,37 @@ public static <T> Optional<T> ignoreEmptyModExceptionWithValue(FreeIpaClientCall
         }
     }
 
+    public static void ignoreEmptyModException(FreeIpaClientRunnable runnable, String message, Object... messageParams)
+            throws FreeIpaClientException {
+        try {
+            runnable.run();
+        } catch (FreeIpaClientException e) {
+            if (isEmptyModlistException(e)) {
+                Optional.ofNullable(message).ifPresentOrElse(
+                        msg -> LOGGER.debug(msg, messageParams),
+                        () -> LOGGER.debug("No modification was needed in FreeIPA is ignored. Exception message: {}", e.getMessage()));
+            } else {
+                throw e;
+            }
+        }
+    }
+
+    public static <T> Optional<T> ignoreDuplicateExceptionWithValue(FreeIpaClientCallable<T> callable, String message, Object... messageParams)
```

Comment:
I don't see this used anywhere. is it needed?

Reply:
I think it was required for an intermittent state. It could come handy in the future :) I wouldn't repush my pr, as it would delay it with at least 1 hour

Reply:
ok. I can see how it could be used and useful later. I agree that it's not worth repushing the pr for